lib/constants/index.ts

```diff
@@ -68,6 +68,9 @@ export const urls = {
   stake: "https://dapp.klimadao.finance/#/stake",
   wrap: "https://dapp.klimadao.finance/#/wrap",
   bond: "https://dapp.klimadao.finance/#/bonds",
+  resources: "/resources",
```

Comment:
lets reserve this object for hardcoded URLs, these relative paths won't work in `dapp`. we can inline for now and later consider a new url object in `site/lib/constants`

Reply:
Good idea! Didn't really think that through.